python/tvm/auto_scheduler/measure_record.py

```diff
@@ -159,3 +175,38 @@ def load_best(filename, workload_key=None, target=None):
             best_res = res
 
     return best_inp, best_res
+
+
+def correct_measure_input(inp, rebuild_state=False):
```

Comment:
The naming is confusing. I imagine the use case would be like
```python
for inp, res in load_records(file):
    inp = correct_measure_input(inp)
```
From the code snippet, it's hard to get what "correct" measure input means.

One suggestion is calling this function in `load_records` by configure, something like `load_records(file, full_model=True)`?

Reply:
This is not used in `load_records`. All existing use cases related to `load_records` do not need this.
This API is used to solve the serialization problem for the No.2 solution in #6650, which is not related to `load_records`

Reply:
I see, but the name is still not quite clear. Maybe `clone_measure_input`, `recover_measure_input`, or something like that?

Reply:
renamed it to `recover_measure_input`